code/game/machinery/machinery.dm

```diff
@@ -603,10 +602,7 @@ obj/machinery/proc/med_scan(mob/living/carbon/human/H, dat, var/list/known_impla
 
 //Damage
 /obj/machinery/proc/take_damage(dam)
-	if(!destructible)
-		return
-
-	if(!dam)
+	if(!dam || CHECK_MULTIPLE_BITFIELDS(resistance_flags, INDESTRUCTIBLE|UNACIDABLE))
```

Comment:
That means that the `machinery`  has to be **both** `INDESTRUCTIBLE` and `UNACIDABLE` for this to pass.
Intended ?